the-lone-dancer.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=The Lone Dancer
+After=basic.target
+
+[Service]
+Type=simple
+#Restart=always
+#RemainAfterExit=yes
+User=root
```

Comment:
Is this necessary?

Reply:
Oh sorry, I was referring to line number 9. Is it necessary to run the script as root?

Reply:
-

Reply:
No, it does not need to be run as root, but it simplifies deployment.

This was a quick-'n-dirty setup. I need to create a new user, though for running the container.

I thought it was standard to run containers as root, but [best practices](https://cloud.google.com/architecture/best-practices-for-operating-containers) says we shouldn't.

Do not merge until this is resolved!